bot/resources/evergreen/py_topics.yaml

```diff
@@ -16,23 +14,34 @@
     - What feature do you think should be added to Python?
     - Has Python helped you in school? If so, how?
     - What was the first thing you created with Python?
+    - What is your favorite Python package?
+    - What standard library module is really underrated?
+    - Have you published any packages on PyPi? If so, what are they?
+    - What are you currently working on in Python?
+    - What's your favorite script and how has it helped you in day to day activities?
+    - When you were first learning, what is something that stumped you?
+    - When you were first learning, what is a resource you wish you had?
+    - What is something you know now that you wish you knew when you were starting out?
+    - What is something simple that you still error on today?
+
+# algos-and-data-structs
+650401909852864553:
+    -
 
 # async
 630504881542791169:
     - Are there any frameworks you wish were async?
     - How have coroutines changed the way you write Python?
+    - What is your favorite async library?
 
 # c-extensions
 728390945384431688:
     -
 
-# computer-science
-650401909852864553:
-    -
-
 # databases
 342318764227821568:
     - Where do you get your best data?
+    - What is your preferred database and for what use?
```

Comment:
Should this use a more precise term in place of database, something like DBMS?

Reply:
I feel like that could start some sort of flame war, so I'll leave it alone for now and see how it does.